app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="w-full flex justify-center items-center h-full">
+  <div class="flex flex-col items-center bg-white w-max">
+
+    <h2 class="font-bold text-secondary text-2xl">Sign up</h2>
+
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: "p-2 my-2 text-lg w-full max-w-sm flex flex-col items-center gap-2" }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="font-bold flex flex-col w-full text-accent items-center">
+        <%= f.label :name %>
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", required: true, class: "rounded-full border-none w-3/4 bg-neutral text-white" %>
+      </div>
+
+      <div class="font-bold flex flex-col w-full text-accent items-center">
+        <%= f.label :email %>
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", required: true, class: "rounded-full border-none w-3/4 bg-neutral text-white w-min" %>
+      </div>
+
+      <div class="font-bold flex flex-col w-full text-accent items-center">
+        <div class="flex">
+          <%= f.label :password %>
+          <% if @minimum_password_length %>
+            <em>(<%= @minimum_password_length %> characters minimum)</em>
+          <% end %>
+        </div>
+        <%= f.password_field :password, autocomplete: "new-password", required: true, class: "rounded-full border-none w-3/4 bg-neutral text-white" %>
+      </div>
+
+      <div class="font-bold flex flex-col w-full text-accent items-center">
+        <%= f.label :password_confirmation %>
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", required: true, class: "rounded-full border-none w-3/4 bg-neutral text-white" %>
+      </div>
+
+      <div class="font-bold bg-secondary hover:bg-secondary-focus px-4 py-1">
+        <%= f.submit "Sign up" %>
+      </div>
+    <% end %>
+
+    <%= render "devise/shared/links" %>
+  </div>
```

Comment:
- [ ] As the requirements say, you design should look exactly the same as the one from the template. Kindly make sure you apply the same here on the login and signup pages, use the same color, fontfamily, font-size, layout,...

Your design | the template 
---|----
![image](https://github.com/stephenkati/Smart_Wallet/assets/65068771/6d6d0f34-2858-410b-91c9-a655d05fe4e4) | ![image](https://github.com/stephenkati/Smart_Wallet/assets/65068771/e831d56c-da3d-459b-ab98-df6e0ac44dde)
![image](https://github.com/stephenkati/Smart_Wallet/assets/65068771/cfb6deef-d93d-478b-aa37-299b24eac067) | ![image](https://github.com/stephenkati/Smart_Wallet/assets/65068771/e1a71bcf-ca74-4a9a-aac7-09a9f3d985cb)